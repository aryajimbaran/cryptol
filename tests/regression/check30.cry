x = foo
  where foo x = x + 5