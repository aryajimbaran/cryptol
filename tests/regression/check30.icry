:l check30.cry
:set warnShadowing=off
:r