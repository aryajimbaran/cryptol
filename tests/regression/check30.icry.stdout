Loading module Cryptol
Loading module Cryptol
Loading module Main
[warning] at ./check30.cry:2:13--2:14
    This binding for x shadows the existing binding from
    (at ./check30.cry:1:1--1:2, Main::x)
Loading module Cryptol
Loading module Main
